src/main/webapp/styles/add-recommendation.css

```diff
@@ -57,7 +57,7 @@
   transition: all 300ms ease-in-out 0s;
 }
 
-/* Add Rec Form Elements */
+/* --- Add Rec Form Elements --- */
 
```

Comment:
do you plan to insert blank line after every comments in css?

Reply:
It's a convention used - so comments /* --- ....... --- */ in this form, with a blank line after, are for a block of different classes, divs etc. whereas /*.....*/ and no line is for the class,div directly underneath, if that makes sense :)